src/components/header/header.tsx

```diff
@@ -0,0 +1,24 @@
+import SearchForm from '../searchForm/searchForm';
+import GenreSelect from '../genreSelect/genreSelect';
+const Header = (props:GenreSelectProps) => {
+
+    const handleSearch = (e: React.FormEvent<HTMLFormElement>)=> {
+        e.preventDefault();
+          console.log(e);    
+        console.log("Search is Working !!");
+    
+      }
+      
+    return (
+        <>
+            <div className="header">
```

Comment:
Please think wisely and double check the components composition. In this Header component you have a div 'header' what ought to be the component wrapper. But. You put 'genreContainer' and 'boerdRectange' here too? Do they really belong to the Header?

Please read this : https://react.dev/learn/thinking-in-react